ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml

```diff
@@ -1,8 +1,24 @@
+base_images:
+  installer:
+    name: installer
+    namespace: ocp
+    tag: latest
```

Comment:
There's no `ocp/installer:latest`. OCP images are collected via version-named imagestreams and tags are individual images in them. Example:

```
name: "4.8"
namespace: ocp
tag: installer
```